docs/pages/build-reference/how-tos.md

```diff
@@ -66,6 +66,7 @@ This is an example of how your package.json might look like:
 
 <ImageSpotlight alt="Secret creation UI filled" src="/static/images/eas-build/environment-secrets/secrets-create-filled.png" />
 
+Before setting up private packages, check the existing configuration (`.npmrc` and `yarnrc.yml`) described on the [Build server infrastructure](/build-reference/infrastructure) page, to verify if it's not affecting your setup.
```

Comment:
```suggestion
Before setting up private packages, check the existing configuration described in the [build server infrastructure](/build-reference/infrastructure.md) page under `.npmrc` and `yarnrc.yml` to verify that it won't affect your setup.
```

I'm guessing that anyone setting this up would understand what will/won't affect their setup? I haven't used private packages so that was my only question when reading this

Reply:
yes, I also updated the first line of the instruction to make it clearer that user needs to know how to configure their project

btw `.md` extensions in link in markdown are not necessary, I'm not sure if they work)

Reply:
they should work (just tested), both live on docs.expo.io as well as clicking to navigate in VSCode